audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -1,7 +1,12 @@
 [
   "anjuta",
+  "anime-downloader",
+  "emscripten",
   "fdroidserver",
   "hive",
+  "joplin-cli",
+  "jupyterlab",
+  "lanraragi",
```

Comment:
I'm not so sure we can add `lanraragi` here.

Reply:
Not sure either, but it seems worked.

Reply:
Sorry, let me clarify: adding `lanraragi` skips the dependency conflict check for `lanraragi`, but it's not clear that we should be skipping that.

Reply:
I get it now. Let's check this later.